src/PKSim.Core/Snapshots/Mappers/ParameterMapper.cs

```diff
@@ -57,6 +75,12 @@ public override Task<SnapshotParameter> MapToSnapshot(IParameter modelParameter)
          if (snapshot.Value == null)
             return parameter;
 
+         var displayUnit = ModelValueFor(snapshot.Unit);
+         if (!parameter.Dimension.HasUnit(displayUnit))
```

Comment:
This can happen either if snapshot is edited by hand OR using some old projects where unit was sometime missing